src/rebar_utils.erl

```diff
@@ -453,6 +453,24 @@ reread_config(ConfigList) ->
                   "and will be ignored.", [])
     end.
 
+%% @doc Given env. variable `FOO' we want to expand all references to
```

Comment:
Has this function just been moved from internal to non-internal code sections without having been exported?

Reply:
Nevermind, just saw it was already exported without having previously been moved.

Reply:
Yeah, that confused me as well, that was the reason for the failing tests above. I exported it but it already was exported it turns out.